tool_result_selection/models.py

```diff
@@ -208,23 +159,41 @@ class Fields(object):
 
         @cached_property
         def datasource_layer_manager(self):
-            return DataSourceLayerManager(self.type.value, self.file_path.value)
-
-        def datasource(self):
-            # TODO: which kind of datasource is this? The netcdf of a
-            # ts_datasources object?
-            return self.datasource_layer_manager.datasource
-
-        def spatialite_cache_filepath(self):
-            return self.datasource_layer_manager.spatialite_cache_filepath
+            """Return DatasourceLayerHelper."""
+            datasource_type = self.type.value
+            if datasource_type != "netcdf-groundwater":
+                pop_up_unkown_datasource_type()
+                raise AssertionError("unknown datasource type: %s" % datasource_type)
+            # Previously, the manager could handle more kinds of datasource
+            # types. If in the future, more kinds again are needed,
+            # instantiate a different kind of manager here.
+            return DatasourceLayerHelper(self.file_path.value)
```

Comment:
Rename the property from `datasource_layer_manager` to `datasource_layer_helper` to be more consistent with the renamed class name.

Reply:
Done